docs/code-reviews/process-guidance/README.md

```diff
@@ -22,6 +22,20 @@ If the team is finding that code reviews are taking a significant time to merge,
 1. Assess the time to merge across sprints to see if the process is improving.
 1. Ping required approvers directly as a reminder.
 
+## Recommendations for a better code review
+
+### Take your time to review but don't go beyond of the specific task
+
+You shouldn't review code too quickly but neither take too long in one sitting. If you have many pull requests or if the complexity of code is demanding the recommendation is to take a break between reviews to recover, and focus on the ones you are most experienced with.
+
+Always remember, the goal of a code review is to verify that the specified task has been achieved. If your suggestions are on code related to, but not included in the PR, raise those suggestions as separate tasks (bugs, technical debt etc.). Don't block the current PR on suggestions outside the scope of the PR.
+
+### Automate whenever possible
```

Comment:
There is already a section on automation in the reviewer guidance (under "Decide on a common standard for each language"). Rather than duplicating this - can we consolidate the information in these sections? I personally think that the consolidated section fits better here than under the reviewer section (as it is general preparation the team needs to do - but we don't want to have duplicate sections)